src/emitters/tests/test_distant.py

```diff
@@ -0,0 +1,119 @@
+import mitsuba
+import pytest
+import enoki as ek
+import numpy as np
+
+
+spectrum_strings = {
+    'd65': """
+        <spectrum version='2.0.0' name="irradiance" type='d65'/>
+        """,
+    'regular': """
+        <spectrum version='2.0.0' name="irradiance" type='regular'>
+           <float name="lambda_min" value="500"/>
+           <float name="lambda_max" value="600"/>
+           <string name="values" value="1, 2"/>
+        </spectrum>
+        """,
+    'null': """
+        <spectrum version='2.0.0' type='uniform'>
+            <float name="value" value="0"/>
+        </spectrum>
+        """,
+}
+
+spectrum_keys = list(set(spectrum_strings.keys()) - {"null"})
+
+
+def make_spectrum(spectrum_key="d65"):
+    from mitsuba.core.xml import load_string
+
+    spectrum = load_string(spectrum_strings[spectrum_key])
+    expanded = spectrum.expand()
+
+    if len(expanded) == 1:
+        spectrum = expanded[0]
+
+    return spectrum
+
+
+def make_emitter(direction="0,0,-1", spectrum_key="d65"):
+    from mitsuba.core.xml import load_string
+
+    if type(direction) is not str:
+        direction = " ".join([str(x) for x in direction])
+
+    return load_string("""
+        <emitter version="2.0.0" type="distant">
+            <vector name="direction" value="{d}"/>
+            {s}
+        </emitter>
+    """.format(d=direction, s=spectrum_strings[spectrum_key]))
+
+
+def test_construct(variant_scalar_rgb):
+    # Test if the emitter can be constructed as intended
+    e = make_emitter()
+    assert not e.bbox().valid()  # Degenerate bounding box
+    assert ek.allclose(
+        e.world_transform().eval(0.).matrix,
+        [[0, 1, 0, 0],
+         [1, 0, 0, 0],
+         [0, 0, -1, 0],
+         [0, 0, 0, 1]]
+    )
+
+
+@pytest.mark.parametrize("spectrum_key", spectrum_keys)
+def test_eval(variant_scalar_spectral, spectrum_key):
+    # Check correctness of the eval() method
+    from mitsuba.render import SurfaceInteraction3f
+
+    direction = np.array([0, 0, -1])
```

Comment:
please use `mitsuba.core.Vector3f` and remove `import numpy as np` in this test file